components/Users/Memberships/index.js

```diff
@@ -380,38 +380,46 @@ const MembershipDetails = ({ userId, membership, ...props }) => {
                   <MoveMembership
                     key='MoveMembership'
                     membership={membership}
-                    refetchQueries={({
-                      data: { moveMembership }
-                    }) => [
-                      {
-                        query: GET_MEMBERSHIPS,
-                        variables: { userId }
-                      }
-                    ]}
+                    refetchQueries={() => [{
+                      query: GET_MEMBERSHIPS,
+                      variables: { userId }
+                    }]}
                   />,
                   !!membership.renew &&
                     <CancelMembership
                       key='CancelMembership'
                       membership={membership}
-                      refetchQueries={() => [
-                        {
-                          query: GET_MEMBERSHIPS,
-                          variables: { userId }
-                        }
-                      ]}
+                      refetchQueries={() => [{
+                        query: GET_MEMBERSHIPS,
+                        variables: { userId }
+                      }]}
                     />,
                   !membership.renew && membership.active &&
                     <ReactivateMembership
                       key='ReactivateMembership'
                       membership={membership}
-                      refetchQueries={({
-                        data: { reactivateMembership }
-                      }) => [
-                        {
-                          query: GET_MEMBERSHIPS,
-                          variables: { userId }
-                        }
-                      ]}
+                      refetchQueries={() => [{
+                        query: GET_MEMBERSHIPS,
+                        variables: { userId }
+                      }]}
+                    />,
+                  (!membership.active || !membership.renew) && membership.type.name === 'MONTHLY_ABO' &&
```

Comment:
IMO we should just have one check to render `ReactivateMembership`

Currently we have:
- `!membership.renew && membership.active`
- `(!membership.active || !membership.renew) && membership.type.name === 'MONTHLY_ABO'`

AFAIK this could be simplified to just the following:
```
!membership.renew && (membership.active || membership.type.name === 'MONTHLY_ABO')
``` 

The props passed to `ReactivateMembership` seem to be the same.

Reply:
True that, adopting.